dev/tools/shared/utils.py

```diff
@@ -41,6 +41,27 @@ def get_full_image_name(args, image_id):
     return f"{image_prefix}{image_id}:{tag}"
 
 
+def update_env_with_go_work_off(kwargs):
+    """ Returns a copy of kwargs with GOWORK=off in the env """
```

Comment:
Docstring/comments should explain "why" but not what. It's easy to understand what this function is doing, so docstring should fill the gap (explain why)